.github/workflows/pull_request.yml

```diff
@@ -0,0 +1,39 @@
+
+name: Tests
+
+# Controls when the action will run.
+on:
+  pull_request:
+  push:
+    branches:
+      - master
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        ansible:
+          - stable-2.9
+          - stable-2.10
+          - stable-2.11
+          - devel
+    steps:
+
+      - uses: actions/checkout@v2
+
+      - name: Installing dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install git+https://github.com/ansible/ansible.git@${{ matrix.ansible }}
```

Comment:
`pip install ansible==2.x.*`?

Reply:
packages renamed upstream to `ansible-base 2.10.*`, `ansible-core 2.11.*`
we could go with non-obvious mapping of ansible<4.0.0 to ansible-base 2.10.* and ansible>=4.0.0 to ansible-core 2.11.*

Reply:
You can still pip install ansible, it comes with batteries